Developer.md

```diff
@@ -2,7 +2,4 @@
 
 Student Information Management System. Post-Secondary Student Financial Aid System.
 
-##IDP TEST USERS BCSC
-1. SS4BPS204 98904 FOUR
-2. SS4BPS202 98902 TWO
-3. SS4BPS209 98909 NIN
+
```

Comment:
Avoid including unnecessary files in the review.
Please do not consider this as a blocker for this review.